.circleci/config.yml

```diff
@@ -216,9 +233,11 @@ jobs:
           command: |
             npm install -g sfdx-cli
             # We need to determine the name of the tarball
-            SFDC_TARBALL=$(ls | grep salesforce*.tgz)
+            SFDC_TARBALL=$(ls | grep salesforce-sfdx-scanner-[0-9]*\\.[0-9]*\\.[0-9]*\\.tgz)
+            # We also ought to determine where we are.
+            DIR=`pwd`
```

Comment:
Can this be `$(pwd)` for consistency? $() has other benefits as well, this is a good discussion about the differences https://stackoverflow.com/questions/9449778/what-is-the-benefit-of-using-instead-of-backticks-in-shell-scripts